### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by dsferruzza**
## Describe your PR

While preparing a future contribution, I stumbled upon some minor issues so here are some fixes!

I also noticed that the README mentions a non-existing `hugo.sh` file; not sure what to do with that.

## Checklist

- [x] I've read the [contributing guidelines](/CleverCloud/documentation/blob/main/CONTRIBUTING.md)

## Reviewers
@juliamrch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,14 @@ To begin your journey with the Clever Cloud Documentation, you need to clone thi
...
+### Install Hugo
+
+Either [install Hugo](https://gohugo.io/installation/) globally in your system or put its [executable](https://github.com/gohugoio/hugo/releases) in the projet's root.
+
 ### Start locally
 
-To run the site in your browser, there is a server built in Hugo you can summon with:  
-`hugo server`  
-A bunch of option a available:
+To run the site in your browser, there is a server built in Hugo you can summon with: `hugo server`.
```
Comment (davlgd) [inline README.md:36]:
```suggestion
To run the site locally, there is a server built in Hugo you can summon with: `hugo server`.
```

**Review by davlgd — APPROVED** [review]
Thanks for this, LGTM

About the `.sh`script, I let @juliamrch check/decide.

**Comment by juliamrch** [comment]
> I also noticed that the README mentions a non-existing `hugo.sh` file; not sure what to do with that.

It's named [clevercloud-deploy-script.sh](https://github.com/CleverCloud/documentation/blob/main/clevercloud-deploy-script.sh) now.

Will review this by tomorrow!

**Review by juliamrch — DISMISSED** [review]
Thanks for the cleanup!

**Review by juliamrch — APPROVED** [review]

**Pull request merged**